css/docs.css

```diff
@@ -63,7 +63,7 @@
   overflow: hidden;
 }
 
-.component-container > div {
+.component-container > div:not(.grid) {
   padding: 2rem 2rem 0;
 }
```

Comment:
can you elaborate on this?

Reply:
yes, sure. here I am selecting all `div` that do not have `grid` class, and are a direct child of element having class `component-container`.